saleor/core/emails.py

```diff
@@ -0,0 +1,14 @@
+from django.contrib.sites.models import Site
+from django.templatetags.static import static
+
+from ..core.utils import build_absolute_uri
+
+
+def get_email_base_context():
+    site = Site.objects.get_current()
+    logo_url = build_absolute_uri(
+        location=None) + static('images/logo-document.svg')
```

Comment:
We should not build urls manually in any case. Instead of `None` you might to pass an actual `static` url to the `build_absolute_uri` function.

`static` will resolve to full absolute url if one uses, for eg. external cloud service to serve the images.
```
In [2]: build_absolute_uri('http://google.pl')
Out[2]: 'http://google.pl'

In [4]: build_absolute_uri(location=None) + 'http://google.pl'
Out[4]: 'http://localhost:8000http://google.pl'
```

Reply:
https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urljoin